### ISSUE TITLE
[Typescript]: Theme values are not updated after modifying them in custom-theme.js, in a Vue TS project.

### ISSUE THREAD

**Issue by jessaleks** [chakra-ui/chakra-ui-vue#296]
**Describe the bug**
After specifying some custom values in `custom-theme.js`, and after using `red.500`, the color that shows up is different from what I specified

**To Reproduce**
1. Start a new Vue project with TS
2. Set up Chakra as you usually would (via @vue/cli)

- My implementation
```ts
...
import Chakra, { CThemeProvider, CReset } from '@chakra-ui/vue';
import * as customTheme from './custom-theme.js';
// tried import {customTheme} from './custom-theme'; as I changed this file to .ts one
// tried exporting a variable as well, and doing import customTheme
...
Vue.use(Chakra, {
	extendTheme: customTheme,
});
...
new Vue({
	router,
	store,
	i18n,
	render: (h) => h(CThemeProvider, [h(CReset), h(App)]),
}).$mount('#app');
```

3. Specify custom values in `custom-theme.js` such as 
```js
export default {
   ...
    colors: {
        // this is essentially blue
        blue: {
            50: '#e9f0f7',
            100: '#d4e0ef',
	    200: '#a9c1df',
            300: '#7da2ce',
            400: '#5283be',
            500: '#2764ae', // the actual blue color
            600: '#1f508b',
            700: '#173c68',
            800: '#102846',
            900: '#081423',
            950: '#040a11'
        },
```

**Expected behavior**
I expect to see my specified value for the blue, not the default chakra one

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox, Brave
 - Version: 80.0.1, 1.13.82 Chromium: 85.0.4183.83

**Comment by codebender828** [comment]
Hey, @jessaleks ! 

I'm not sure I understood the exact cause of the problem that you are facing. Could you provide a codesandbox reproducing the problem or a GitHub repo reproducing issue?

This will help determine how to best debug it.

**Comment by jessaleks** [comment]
@codebender828 sure! 

(Repo)[https://github.com/jessaleks/debug-chakra-app]
Here you go

**Comment by codebender828** [comment]
Hey @jessaleks.

Thank you. I just checked the repo you shared. It doesn't look like it has the code you want to reproduce. Could you double check to make sure?

**Comment by jessaleks** [comment]
@codebender828 sorry, it should have everything right now. 
It's a minimal app, however the problem still persists.

**Comment by codebender828** [comment]
Thanks for the repo link, @jessaleks . Unfortunately I haven't been able to reproduce this issue [here](https://github.com/codebender828/chakra-issue-merge-theme) in a JS environment. (Deployed [here](https://debug-chakra-merge-theme.vercel.app)) However I was able to reproduce it when I cloned your project. I'm not really sure why.

I currently have a limited knowledge of typescript to be able to give you a direct path to resolve this issue. 

I'll open this up for contribution. I hope you're able to find a solution soon.
